src/api/app/controllers/source_controller.rb

```diff
@@ -395,8 +395,8 @@ def lastevents_public
   def lastevents
     path = get_request_path
```

Comment:
@hennevogel you could even get rid of this line, because the `pass_to_backend` method defaults to `get_request_path` if the path you pass is nil. But not a blocker